spec/shared_examples/confirmation_email.rb

```diff
@@ -2,7 +2,7 @@
 
 RSpec.shared_examples 'a confirmation email sender' do
   it 'sends an email' do
-    expect(UsersMailer.deliveries.size).to eq(1)
```

Comment:
Please don't delete this check. I'd still like to know if all emails have actually been delivered.